graph/test/tarjan.test.ts

```diff
@@ -0,0 +1,78 @@
+import { tarjan } from "../tarjan";
+
+describe("tarjan", () => {
```

Comment:
Aren't these tests the same as for Kosaraju's algorithm? I would suggest deduplicating them, unless you want to check the topological order.

(BTW, it just occurred to me that the Kosaraju tests should ideally not care about the order of the strongly connected components...)

Reply:
> Aren't these tests the same as for Kosaraju's algorithm? I would suggest deduplicating them, unless you want to check the topological order.

Yes they are the same. Yeah I should have probably wrote some shared code between the two. But since you mentioned topological sort, the order does actually matter. Kosajaru should have topological sort (see comment above). So I think we can keep the tests here and in Kosajaru as is.